lib/plug/debugger.ex

```diff
@@ -122,7 +122,23 @@ defmodule Plug.Debugger do
 
       def call(conn, opts) do
         try do
-          super(conn, opts)
+          case conn do
+            %Plug.Conn{path_info: ["__debug__", "action"], params: params} ->
+              %Plug.Conn{params: params} = Plug.Parsers.call(conn, Plug.Parsers.init(parsers: [:urlencoded]))
+
+              {module, function, args} = 
+                params["encoded_action"] 
+                |> Base.decode64!() 
+                |> :erlang.binary_to_term()
```

Comment:
Please use `Plug.Crypto.safe_binary_to_term` which does not allow serialized functions. :)